cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sources/StreamBatchSource.java

```diff
@@ -119,17 +131,6 @@ public void configurePipeline(ETLStage stageConfig, PipelineConfigurer pipelineC
     } else {
       this.formatSpec = null;
```

Comment:
there is no need for formatSpec to be a class variable, should be local to this method.